src/modules/networkcfg/main.py

```diff
@@ -49,7 +52,10 @@ def run():
             try:
                 shutil.copy(source_network, target_network)
             except FileNotFoundError:
-                libcalamares.utils.debug("Can't copy network configuration files in {}".format(source_network))
+                libcalamares.utils.debug(
+                    "Can't copy network configuration files in",
+                    "{}".format(source_network)
+                    )
```

Comment:
`libcalamares.utils.debug()` only takes one parameter; that `,` should be a +, and you've dropped a space in the string.